packages/paste-website/src/pages/components/toggle/index.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Toggle
+package: '@twilio-paste/toggle'
+description: A toggle is an interactive binary switch.
+slug: /components/toggle/
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Avatar} from '@twilio-paste/avatar';
+import {Box} from '@twilio-paste/box';
+import {Toggle} from '@twilio-paste/toggle';
+import Changelog from '@twilio-paste/chat-log/CHANGELOG.md';
+import {HelpText} from '@twilio-paste/help-text';
+import {useUID} from '@twilio-paste/uid-library';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {basicToggle, onToggle, disabledToggle} from '../../../component-examples/ToggleExamples.ts';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/toggle"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/toggle/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+        description
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Toggle"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/toggle"
+  storybookUrl="/?path=/story/components-toggle--default"
+  data={props.data}
+  description={props.data.mdx.frontmatter.description}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Toggle
+
+A Toggle is an interactive binary switch. It should be used in forms when an "on/off" or "yes/no" input is needed. Use label text to describe the purpose of the Toggle. The label text shouldn't change when the state of the Toggle changes. Help Text can also be used to provide additional guidance.
+
+### Accessibility
+
+Toggle has `role="switch"`. The `aria-labelledby` attribute is automatically set on the component, which ensures that assisstive technology devices can read the Toggle's label. When the Toggle is on, `aria-selected` is set to `"true"` (it defaults to `"false"`).
```

Comment:
Issue: it is `aria-checked`, not `aria-selected`

Reply:
thank you!

Reply:
suggestion: this section would be more clear without the parenthetical
```suggestion
Toggle has `role="switch"`. The `aria-labelledby` attribute is automatically set on the component, which ensures that assistive technology devices can read the Toggle's label. The 'aria-checked' attribute defaults to `"false"`, and is set to `"true"` when the Toggle is on.
```